.github/workflows/build.yml

```diff
@@ -11,6 +11,45 @@ on:
 # We build for other platforms except linux which is already covered in build-and-test.
 # Also, We're not running tests here as those are already covered with linux build.
 jobs:
+  Get-CI-Image-Tag:
+    uses: opensearch-project/opensearch-build/.github/workflows/get-ci-image-tag.yml@main
+    with:
+      product: opensearch
+
+  build-linux:
+    strategy:
+      matrix:
+        java: [11, 17]
+
+    name: Build and Test CCR Plugin on Linux
+    runs-on: ubuntu-latest
+    needs: Get-CI-Image-Tag
```

Comment:
Not sure if we're adding any value by this extra workflow `Build on ubuntu with ci image`. We're already testing the same under `build-and-test`. This one can be removed.